src/SharedLibrary/IntegrationTests/SharedLibrary_SSWFeaturizer_IntegrationTest.cpp

```diff
@@ -0,0 +1,84 @@
+// ----------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License
+// ----------------------------------------------------------------------
+#define CATCH_CONFIG_MAIN
+#include "catch.hpp"
+
+#include "GeneratedCode/SharedLibraryTests_StandardScaleWrapperFeaturizer.h"
+
+TEST_CASE("int8_t_with_mean_with_std") {
+    StandardScaleWrapperFeaturizer_int8_Test(
+        std::vector<std::int8_t>{
+            static_cast<std::int8_t>(0),
+            static_cast<std::int8_t>(0),
+            static_cast<std::int8_t>(1),
+            static_cast<std::int8_t>(1)
+        },
+        std::vector<std::int8_t>{
+            static_cast<std::int8_t>(2)
+        },
+        [](std::vector<std::double_t> const &args) {
+            return args == std::vector<std::double_t>{static_cast<std::double_t>(3)};

```

Comment:
>static_cast<std::double_t>(3) [](start = 54, length = 29)

would 3.0 be a bit more readable?

Reply:
3.0 is a better way to write it.
=={3.0} is not a valid expression since initializer list cannot be used on the right hand side of operator '=='.

Reply:
Creating a container to compare a single element is not a good choice.

---
In reply to: [366543401](https://github.com/microsoft/FeaturizersLibrary/pull/65#discussion_r366543401) [](ancestors = 366543401)